sihl/user/service.ml

```diff
@@ -144,22 +144,22 @@ module Make (Repo : Sig.REPOSITORY) : Sig.SERVICE = struct
     match
       User.validate_new_password ~password ~password_confirmation ~password_policy
     with
-    | Error msg -> Lwt_result.fail msg
+    | Error msg -> Lwt_result.fail @@ Model.Error.Other msg
```

Comment:
This should be InvalidPasswordProvided or something like that

Reply:
Fixed!